content/_index.md

```diff
@@ -12,56 +12,60 @@ Congratulations on being part of the AVADO community !
 
 We have bundled some resources for you with answers to common questions that people have with regards to setting up and running an AVADO node.
 
-Please use the navigation menu on the left to browse the docs.
+<!-- Please use the navigation menu on the left to browse the docs. -->
 
-## Overview
+<!-- ## Overview -->
 
 {{< columns >}}
 
-### Get Started
+## Getting Started
 
-Check the [Getting started](getting-started) page to get started
+[Getting started with your AVADO](getting-started)
 
 <--->
 
-### Shipping, delivery and returns
+## Video's
 
-Check our [AVADO delivery](/support/delivery) page for more info.
+We host a number of [YouTube video's](https://www.youtube.com/avadocloud) that go through the setup of certain packages step by step
 
 <--->
 
-### Youtube
+## Support
 
-We host a number of [YouTube video's](https://www.youtube.com/avadocloud) that go through the setup of certain packages step by step
+- [Ask for Help on Telegram]({{< relref "/support/telegram" >}})
```

Comment:
```suggestion
[Ask for Help on Telegram]({{< relref "/support/telegram" >}})  
```
The dots look ugly here
<img width="990" alt="image" src="https://user-images.githubusercontent.com/735288/200857339-84bb5ff7-1d9d-4287-b34e-e0733f54300b.png">